libs/ui/inputs/src/lib/copy-text-button/copy-text-button.component.html

```diff
@@ -1,11 +1,18 @@
 <div class="flex">
   <input
-    *ngIf="displayText"
+    *ngIf="displayText && rows <= 1"
     type="text"
-    class="grow py-2 px-4 bg-gray-200 opacity-50 rounded-l-md border border-gray-300"
+    class="grow py-2 px-4 bg-gray-200 opacity-50 rounded-l-md border border-gray-300 truncate"
     value="{{ text }}"
     readonly
   />
+  <textarea
+    *ngIf="displayText && rows > 1"
+    [rows]="rows"
+    class="grow py-2 px-4 bg-gray-200 opacity-50 rounded-l-md border border-gray-300 overflow-y-hidden h-max"
```

Comment:
```suggestion
    class="grow py-2 px-4 bg-gray-200 opacity-50 rounded-l-md border border-gray-300 overflow-y-auto h-max"
```

Otherwise there is no way to read the whole text without copypasting or expanding the text area.

Reply:
I prefer the existing option of expanding the text area in the lower right corner. Don't you think it is sufficient? The vertical scrollbar appears pretty ugly to me. We could also raise the number of rows, but this might become too intrusive.

Reply:
The scroll bar makes sense I guess, scroll bars are always ugly but not being able to scroll really struck me as an issue. Also think of keyboard accessibility ;)

Reply:
Ok, I added the scrollbar for the sake of accessibility ;-)